### PR TITLE
chore: Add Issue to public roadmap project

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: .github/workflows/issue_to_project.yml [hunk 1]
```diff
@@ -23,3 +23,11 @@ jobs:
...
           resource_node_id: ${{ github.event.issue.node_id }}
           status_value: ${{ env.todo }} # Target status
+      - name: Add issue to Public Roadmap
+        uses: leonsteinhaeuser/project-beta-automations@v2.0.1
+        with:
+          gh_token: ${{ secrets.GH_CQ_BOT }}
+          organization: cloudquery
+          project_id: 7
+          resource_node_id: ${{ github.event.issue.node_id }}
+          status_value: No Status
```
Comment (erezrokah) [inline .github/workflows/issue_to_project.yml:33]: I'm not sure `No Status` will work as it has to be one of:
https://github.com/orgs/cloudquery/projects/7/settings/fields/Status
image

Unless I'm missing something and there's another status field.
How about we drop `status_value` as it's not required

**Review by erezrokah — CHANGES_REQUESTED** [review]
Looks good pending 1 comment. Also can we remove the other triaging step?

File: .github/workflows/issue_to_project.yml (hunk 1 above)
Comment (bbernays) [reply .github/workflows/issue_to_project.yml]: I am good with that

**Comment by erezrokah** [comment]
@bbernays WDYT about removing the `- name: Move issue to ${{ env.todo }}` step? I'm not sure we need to add issues to 2 boards

**Comment by bbernays** [comment]
> @bbernays WDYT about removing the `- name: Move issue to ${{ env.todo }}` step? I'm not sure we need to add issues to 2 boards

Sorry I had the changes local and forgot to push them

**Review by erezrokah — APPROVED** [review]

**Pull request merged**